app/src/main/res/values-nl/strings.xml

```diff
@@ -95,10 +100,9 @@
     <string name="scan_qr_code">Scan QR code</string>
     <string name="show_qr_code">Toon QR code</string>
     <string name="contact_set_name">Geef contact een naam</string>
-    <string name="contact_exists_message">Contact bestaat al, geef aub een unieke naam</string>
+    <string name="contact_exists_give_unique_name">Contact bestaat al, geef aub een unieke naam</string>
```

Comment:
Is al uit een oude PR zie ik, maar kun je 'aub' veranderen in "alstublieft" of "a.u.b."